doc/fileformat/nmodl.rst

```diff
@@ -109,3 +109,35 @@ Arbor-specific features
     POST_EVENT(t) {
        g = g + (0.1*t)
     }
+
+Nernst
+------
+Many mechanisms make use of the reversal potential of an ion (``eX`` for ion ``X``).
+A popular equation for determining the reversal potential during the simulation is
+the `Nernst equation <https://en.wikipedia.org/wiki/Nernst_equation>`_.
+Both Arbor and NEURON make use of ``nernst``. Arbor implements it as a mechanism and
+NEURON implements it as a built-in method. However, the conditions for using the
+``nernst`` equation to change the reversal potential of an ion differ between the
+two simulators.
+
+1. In Arbor, the reversal potential of an ion remains equal to its initial value (which
+has to be set by the user) over the entire course of the simulation, unless another
+mechanism which alters that reversal potential is explictly added to the model
+(see :ref:`cppcablecell-revpot` for details).
+
+2. In NEURON, there is a rule which is evaluated under the hood to determine whether or
+not the reversal potential of an ion remains constant or is calculated using ``nernst``.
+The rule is documented `here <https://neuron.yale.edu/neuron/static/new_doc/modelspec/programmatic/ions.html>`_
+and can be summarized as follows:
+
+  If the internal or external concentration of an ion is **written**, and its reversal
+  potential is **read but not written**, then the nernst equation is used **continuously**
+  during the simulation to update the reversal potential of the ion.
+  And if the internal or external concentration of an ion is **read**, and its reversal
```

Comment:
Do we have a way of saying that in Arbor?

Reply:
The Arbor policy is stated above and is simpler. Reversal potentials of an ion remain at the user supplied initial value, or they are altered by a mechanism. The link provided gives more details on how to create/use such a mechanism.

Reply:
I do not debate the simplicity nor the wisdom of Arbor's policy here, but I expect people coming from Neuron
wanting to re-create what they know (again: no judgement), so offering help in this direction is in the direct
interest of our users.

Reply:
What kind of help are you thinking? Is the info in (1) missing something?

Reply:
Something like an example maybe, how one would query the `cable_cell` or `decor` for the relevant mechanisms
and their properties via `mech_info` to decide whether Neuron would have used `nernst`.

That would require some infrastructure, as we need to interrogate the `cable_cell` object whether two regions
overlap (we need the morphology and possibly even the discretised version here). Depending on the final 
judgement on Neurons behaviour (see below) that is.

Reply:
This is a bit out of the scope of this PR in my opinion.
Also, it is good to know what NEURON does, but I don't think we should encourage it by showing how to replicate it in Arbor.

Reply:
I partially agree about the scope issue. However, for verification/reproduction it seems important to have this somewhere. Similarly, for onboarding new users who actively seek a Neuron-like experience (again, not judging) it might be convenient to.